mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -242,7 +243,9 @@ def action_manual_check_mailgun(self):
                 raise ValidationError(_("Event information not longer stored"))
             for item in content["items"]:
                 if not self.env['mail.tracking.event'].search(
-                        [('mailgun_id', '=', item["id"])]):
+                        [('mailgun_id', '=', item["id"])]) and (
+                            item.get("recipient", "")
+                            == email_split(tracking.recipient)[0]):
```

Comment:
What happen if you have more than one email in the same field? Shouldn't you process both?

Reply:
As far as I know there is a tracking per recipient. I use `email_split` to extract the address in cases like **"Someone" <someone@example.com>**